pkg/cli/debug/debug.go

```diff
@@ -1087,3 +1095,38 @@ func setNodeName(template *corev1.PodTemplateSpec, nodeName string, overrideWhen
 	}
 	return template
 }
+
+func (o *DebugOptions) resolveImageStreamTagString(s string) (string, error) {
+	namespace, name, tag := parseImageStreamTagString(s)
```

Comment:
Use `imageutil.ParseImageStreamTagName` from library-go.

Reply:
`imageutil.ParseImageStreamTagName` does not take namespace into account, not usable here

Reply:
Good point.